lib/Service/ParticipantService.php

```diff
@@ -2382,10 +2382,13 @@ public function getParticipantByActor(Room $room, string $actorType, string $act
 		return $participant;
 	}
 
-	public function setHasScheduledMessages(Participant $participant, bool $hasScheduledMessages): void {
+	public function setHasScheduledMessages(Participant $participant, int $hasScheduledMessages): void {
 		$attendee = $participant->getAttendee();
+		if ($attendee->getHasScheduledMessages() === Participant::ERROR_SCHEDULED_MESSAGE) {
```

Comment:
How is this "unset" after a user resolved a failed scheduled message by deleting/editing it?